jujugui/static/gui/src/app/init/component-renderers-mixin.js

```diff
@@ -232,14 +232,7 @@ Browser: ${navigator.userAgent}`
     };
     const githubIssueLink =
       `${githubIssueHref}?${queryString.stringify(githubIssueValues)}`;
-    const address = function() {
-      if (db.environment.get('jujushellURL')) {
-        return `ws://${db.environment.get('jujushellURL')}/ws/`;
-      }
-      if (config.jujushellURL) {
-        return config.jujushellURL;
-      }
-    }();
+    const address = initUtils.jujushellURL(localStorage, db, config);
     if (!address) {
```

Comment:
should this not be more explicit in this error? If we don't have an address we know what's up and the generic "unknown error" seems confusing?

Reply:
I made this clearer.